src/components/select-utils.js

```diff
@@ -38,10 +38,12 @@ export const joinPostData = state => postId => {
     if (author === placeholderUser) {
       console.log('We\'ve got comment with unknown author with id', placeholderUser.id);
     }
+    const previousAuthor = (comments[index-1] || {}).user;
+    const omitBubble = author === previousAuthor;
```

Comment:
Is there possible any sort of race condition when some user's property is changed and they are not equal anymore? Could we just compare their IDs?